introduction.md

```diff
@@ -1,2 +1,7 @@
 Hello, I am Meru, working as BA/PM in Germany-FFM and doing some coding in my sparetime.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?